libraries/AP_Scripting/AP_Scripting.h

```diff
@@ -20,6 +20,10 @@
 #include <AP_Param/AP_Param.h>
 #include <GCS_MAVLink/GCS.h>
 #include <AP_Filesystem/AP_Filesystem.h>
+#include <AP_HAL/I2CDevice.h>
+
+#define SCRIPTING_MAX_NUM_I2C_DEVICE 4
```

Comment:
Maybe add a `#ifndef` check before this, that way a hwdef can override it?

Reply:
done